sysutils/glogg/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           boost                           1.0
+PortGroup           compiler_blacklist_versions     1.0
+PortGroup           github                          1.0
+PortGroup           qmake5                          1.0
+
+github.setup        nickbnf glogg 1.1.4 v
+github.tarball_from archive
+revision            0
+
+description         A fast, advanced log explorer
+
+long_description    \
+    ${name} is a multi-platform GUI application that helps browse and search \
+    through long and complex log files. It is designed with programmers and \
+    system administrators in mind and can be seen as a graphical, interactive \
+    combination of grep and less.
+
+categories          sysutils
+installs_libs       no
+license             GPL-3
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  3e68f02c65a863287f244782596c6473425f6d3e \
+                    sha256  1226b726317f42a52caa708852d40f59665dbc7d96e2520aff7640c373c67ce8 \
+                    size    1194722
+
+qt5.min_version     5.2
+
+compiler.blacklist-append   *clang*
```

Comment:
This was still a work in progress, but it's because `glogg` only wants to build with `gcc`: https://github.com/nickbnf/glogg#requirements

Needs to be revisited though.